### DISCUSSION TITLE
E-mail to translate@zettlr.com is bounced back

### DISCUSSION THREAD

**Discussion by wiwikuan** [Zettlr/Zettlr#3091] [General]
Hi, I'm from Taiwan, and my country's language "Traditional Chinese" is not on the translate list. I would like to help translate.

I wrote to translate@zettlr.com but my e-mail is bounced back, maybe there's problem with that e-mail address?

**Comment by nathanlesage** [comment]
First, I would like to apologize. It has taken me 31 years (until today) to understand the difference between traditional and simplified Chinese. For that I would like to apologize.

Then, I just added the `zh-TW` language on Translate (+ I will look into the email issue). In line with the standard on Zettlr Translate, I have called it "Chinese (Taiwan)".

However, I thought whether that could be politically problematic for people living in mainland China who want to use the app in traditional Chinese? Since you are affected, what would you think is the most clever approach here? "Traditional Chinese" or is "Chinese (Taiwan)" fine?

**Reply by wiwikuan** [reply]
As a Taiwanese, I definitely prefer "Chinese (Taiwan)"; But if you really really afraid of people from China getting angry (and they will), "Traditional Chinese" is OK, too.

Just don't use "Taiwan, China" like Typora does on their website, that's disgusting. Actually this is why I switched from Typora and found Zettlr in the first place. :)

**Reply by nathanlesage** [reply]
Alright, thank you for clarifying! I'll change the language name to Traditional Chinese according to that.

I don't understand why Typora would imply Taiwan is part of China when it clearly isn‘t … (and I understand why you find that disgusting)

**Reply by wiwikuan** [reply]
Thanks a lot for that! I just finished all the translations.

**Reply by nathanlesage** [reply]
Whoa, awesome! Thank you :)
